### ISSUE TITLE
<spc> s s and <spc> s S behavior

### ISSUE THREAD

**Issue by sutram** [VSpaceCode/VSpaceCode#75]
Hi there,

I am using version 0.7.1 of VSpaceCode. I tried this on Ubuntu 20.04 and OS X 10.15.6

I am unable to make these two commands work properly. 

<spc> s s doesn't do anything
<spc> s S throws an error: TaskQueue: Error running task. Failed to handle key= . command 'fuzzySearch.activeTextEditorWithCurrentSelection' not found.

Thanks,

**Comment by stevenguh** [comment]
Make sure `jacobdufault.fuzzy-search` extension is installed. It should be automatically installed by vscode since this extension is an extension pack that bundles that; however, it might not.

Also, is there any chance you are running vscode remote?

**Comment by stevenguh** [comment]
I just realized is a bug I introduced with `<spc> s s` as it didn't call the right command

**Comment by stevenguh** [comment]
Released `v0.7.2` to fix the issue where `<spc> s s` wasn't calling the right command.

**Comment by sutram** [comment]
@stevenguh That was a quick release! 😄 

I tried 0.7.2 and now with <spc> s s, I get the same error: TaskQueue: Error running task. Failed to handle key= . command 'fuzzySearch.activeTextEditorWithCurrentSelection' not found.

<spc> s S also results in the same error: TaskQueue: Error running task. Failed to handle key= . command 'fuzzySearch.activeTextEditorWithCurrentSelection' not found.

I checked and I have fuzzy-search 0.0.2 by jacobdufault installed

Thanks!

**Comment by sutram** [comment]
@stevenguh Sorry, also forgot to answer your other question - I am not editing a remote file. This is a local file

**Comment by stevenguh** [comment]
Can you try using the command palette (`Ctrl+Shift+P`) to search `fuzzySearch.activeTextEditorWithCurrentSelection` to see if you can see and trigger the command manually.
![image](https://user-images.githubusercontent.com/3455662/88353416-36c1d700-cd12-11ea-83b2-4174879df9ac.png)

Also, have you tried restarting/reloading vscode?

**Comment by sutram** [comment]
@stevenguh I did restart vscode. Here is the interesting thing:

From the command palette, I only see "Fuzzy outline" and if I go and check the fuzzy extension, the only command that is exposed by that extension is fuzzySearch.activeTextEditor with a description of "Fuzzy outline"

Thanks

**Comment by stevenguh** [comment]
This is very interesting. I figured it out, apparently for some reason VSCode cached a development version on my local vscode instance which contains that command. The current latest version `v0.0.2` of fuzzy search doesn't contain that command. I opened an issue in the repo, and hopefully we will get a new release soon. Worst case, we might have to fork the repo and release it under VSpaceCode org.
https://github.com/jacobdufault/vscode-fuzzy-search/issues/11

Meanwhile, merge this into your `settings.json` to override the default command
```jsonc
{
  "vspacecode.bindingOverrides": [
    {
      "keys": "s.s",
      "name": "Fuzzy search",
      "type": "command",
      "command":"fuzzySearch.activeTextEditor",
    }
  ]
}
```
